deepdoc/parser/figure_parser.py

```
@@ -46,33 +46,38 @@ def vision_figure_parser_figure_data_wrapper(figures_data_without_positions):
 
 
-def vision_figure_parser_docx_wrapper(sections, tbls, callback=None, **kwargs):
+def vision_figure_parser_docx_wrapper(sections, tbls, callback=None, lang="English", **kwargs):
     if not sections:
         return tbls
     try:
         vision_model_config = get_tenant_default_model_by_type(kwargs["tenant_id"], LLMType.VISION)
-        vision_model = LLMBundle(kwargs["tenant_id"], vision_model_config)
+        vision_model = LLMBundle(kwargs["tenant_id"], vision_model_config, lang=lang)
         callback(0.7, "Visual model detected. Attempting to enhance figure extraction...")
     except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)